iosApp/iosApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,62 +6,62 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/google/abseil-cpp-binary.git",
       "state" : {
-        "revision" : "748c7837511d0e6a507737353af268484e1745e2",
```

Comment:
question: are these package resolution changes expected?

Reply:
I think this is the project lockfile instead of the workspace lockfile, and we usually change the workspace lockfile but not the project lockfile. (Why there are two of them that are maintained independently is anyone's guess.)

Reply:
I am also not sure why there is two, these changes look like they're correct though just based on the version updates from when I bumped AppCheck